common/src/main/java/io/pravega/common/function/Callbacks.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
```

Comment:
This is my auto-formatter. It auto-formats all lines of code that have been touched. Proper JavaDoc is a bit HTML-like, hence the additions of all the paragraph tags.